drivers/ethernet/phy/phy_microchip_t1s.c

```diff
@@ -533,6 +612,13 @@ static int phy_mc_t1s_init(const struct device *dev)
 			return ret;
 		}
 		break;
+	case PHY_ID_LAN867X_REVD0:
+		ret = phy_mc_lan867x_revd0_config_init(dev);
+		if (ret) {
+			LOG_ERR("PHY initial configuration error: %d\n", ret);
+			return ret;
+		}
```

Comment:
might be a good idea to move this part out of the switch case, as this is the same, for all cases except the default